apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResource.java

```diff
@@ -451,7 +596,7 @@ private Integer countRecords() {
       Set<String> uniqueIds = new HashSet<String>();
       csvParser.forEach(
           csvRecord -> {
-            String id = csvRecord.get(idColumn);
+            String id = idColumn == null ? UUID.randomUUID().toString() : csvRecord.get(idColumn);
```

Comment:
I looked into this and I was wrong. This is actually needed for the bene history file which has a null id column (see RifFileType.java, line 7). Without this code you get an NPE in RifLoaderIT line 562 when it tries to calculate the number of records in the bene history file.

Reply:
Hmm, that's definitely an oversight that should be fixed -- this code hasn't needed to support bene history until now.

Could you perhaps make `RifFileType.getIdColumn()` return an `Optional`, instead of a bare `null`, and have this method check for that? When it is `Optional.empty()`, just have the method fall back to counting lines, instead of IDs.

That make sense?

Reply:
If not, please just add a FIXME comment to this line explaining why the null check is necessary, and I'll address it in the other PR, myself.

Reply:
Made RifFileType.idColumn an Optional as suggested.